src/integrationTest/java/org/opensearch/test/framework/cluster/TestRestClient.java

```diff
@@ -167,6 +168,13 @@ public HttpResponse patch(String path, String body) {
 		return executeRequest(uriRequest, CONTENT_TYPE_JSON);
 	}
 
+	public HttpResponse assignRoleToUser(String username, String roleName) {
+		Objects.requireNonNull(roleName, "Role name is required");
+		Objects.requireNonNull(username, "User name is required");
+		String body = String.format("[{\"op\":\"add\",\"path\":\"/opendistro_security_roles\",\"value\":[\"%s\"]}]", roleName);
```

Comment:
Shouldn't this contain `opensearch_security_roles`?

Reply:
Sorry, I did not understand your comment :( Could you elaborate, please?

Reply:
My bad, I meant that since we are no longer using `opendistro` convention should this initialization not contain `opendistro` and use `opensearch` instead. Let me know if I need to elaborate more

Reply:
Actually, the older name is still used in API, probably due to backwards compatibility, please see https://opensearch.org/docs/1.2/security-plugin/access-control/api#patch-user

Reply:
You are correct. It is still the old name: https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java#L109